cypress/integration/specs/home-spec.ts

```diff
@@ -8,23 +8,29 @@ describe('Homepage test', () => {
     cy.visit('/');
   });
 
-  it('should render the homepage', () => {
-    cy.contains('Gallery', { matchCase: false }).should('be.exist');
-  });
+  context('Homepage', () => {
+    it('should render the homepage', () => {
+      cy.visit('/');
+      cy.contains('Gallery', { matchCase: false }).should('be.exist');
+    });
 
-  it('should redirect to members page when click the explore button', () => {
-    home.getExploreButton().should('be.exist');
-    home.getExploreButton().click();
-    cy.url().should('include', '/home');
-  });
+    it('should redirect to members page when click the explore button', () => {
+      home.getExploreButton().should('be.exist');
+      home.getExploreButton().click({
+        force: true,
+      });
 
-  // TODO: reenable when synpress `acceptMetamaskAccessRequest()` bug is fixed!
-  // it('should redirect to home page when click the sign in button', () => {
-  //   home.getSignInButton().should('be.exist');
-  //   home.getSignInButton().click();
-  //   home.getEthereumButton().click();
-  //   home.getMetaMaskButton().click();
-  //   home.acceptMetamaskAccessRequest();
-  //   cy.url().should('include', `/home`);
-  // });
+      cy.url().should('include', '/home');
+    });
+
+    // TODO: reenable when synpress `acceptMetamaskAccessRequest()` bug is fixed!
```

Comment:
is there still a problem with clicking the button?

Reply:
Unfortunately yes 😞 . its intermittent issue